test/TestUtilities/Test.Utility/MockServer.cs

```diff
@@ -26,9 +26,12 @@ public class MockServer : IDisposable
     {
         private Task _listenerTask;
         private bool _disposed = false;
+        private AuthenticationSchemes _authenticationSchemes;
+#pragma warning disable CA2213 // Disposable fields should be disposed
+        private HttpListener _listener;
+#pragma warning restore CA2213 // Disposable fields should be disposed
```

Comment:
This method already has a `Dispose` method, which you're using to set `_listener = null`.  Is there a reason you're not calling `_listener.Dispose()` before setting the field to null?

Reply:
For some reason `HttpListener` does not have a `Dispose()` method?  Since its just test code I suppressed the warning.  I've pushed a commit to cast it to `IDisposable` instead which appears to make things happy.